### PR TITLE
v1.8 backport 2020-09-18

### PR DESCRIPTION
**Pull request by kkourt**
v1.8 backports 2020-09-18

 * #12675 -- fix(12664): initialize gops in RootCmd execution function (@fristonio)
 * #13101 -- docs: Document restart-ginkgo CI-trigger phrase (@pchaigno)
 * #13141 -- operator: fix invocation with `--help` option (@tklauser)
 * #13044 -- contrib: Add release helper scripts for preparing micro releases (@joestringer)
      - conflict resolved: change on slightly different text (see commit message)
 * #13177 -- docs: Update kernel requirements for advanced features (@pchaigno)
      - removed bandwidth-manager requirement
 * #13190 -- service: Fix panic when restoring services with enable-health-check-nodeport: false (@gandro)
 * #13188 -- daemon: report HTTP service unavailable (503) in unhealthy state (@tklauser)
 * #13175 -- agent: Add CILIUM_ENABLE_POLICY env into the helm chart. (@camilo-schoeningh-sociomantic)
 * #12925 -- endpoint: Update proxy policy after adding redirects (@jrajahalme)
      - conflict resolved: seemingly unrelated file change omitted (see commit message)
 * #13197 -- ipam: Warn when failing to update CN status (@christarazi)
 * #13205 -- identity: Fix user-space security identity lookup in cluster mesh (@gandro)

Once this PR is merged, you can update the PR labels via:
```upstream-prs
$ for pr in 12675 13101 13141 13044 13177 13190 13188 13175 12925 13197 13205; do contrib/backporting/set-labels.py $pr done 1.8; done
```

### CODE REVIEW

**Review by tklauser — APPROVED** [review]
LGTM for my changes

File: Documentation/install/system_requirements.rst [hunk 1]
```diff
@@ -151,22 +151,24 @@ by adding the following to the helm configuration command line:
...
+
+============================= ===============================
+Cilium Feature                Minimum Kernel Version
+============================= ===============================
+:ref:`concepts_fragmentation` >= 4.10
+:ref:`cidr_limitations`       >= 4.11
+:ref:`host-services`          >= 4.19.57, >= 5.1.16,  >= 5.2
+:ref:`kubeproxy-free`         >= 4.19.57, >= 5.1.16,  >= 5.2
+:ref:`session-affinity`       >= 5.7
+BPF-based proxy redirection   >= 5.7
```
Comment (pchaigno) [inline Documentation/install/system_requirements.rst]: That line should also be removed. It's a v1.9 feature.

**Review by pchaigno — CHANGES_REQUESTED** [review]
One other line to remove from documentation below. Otherwise LGTM!

File: Documentation/install/system_requirements.rst (hunk 1 above)
Comment (kkourt) [reply Documentation/install/system_requirements.rst]: Thanks for catching this! Pushed a new version.

**Review by pchaigno — APPROVED** [review]
:+1: for my changes!

**Comment by tklauser** [comment]
test-backport-1.8

**Review by fristonio — APPROVED** [review]
LGTM for my changes. :rocket:

**Review by christarazi — APPROVED** [review]

**Pull request merged**